lib/live/form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative 'view'
+require 'trenni/builder'
+require 'json'
+
+module Live 
+    #Represents a Form within the view layer
+    #Differs from Live::View because the server sends both the 
+    #details and a serialized form from the view layer.
+    class Form < Live::View 
+        
+        #returns [String] javascript code to execute when the form is submitted.
+        def handleForm(details=false)
+            if details 
+                return "live.handleForm(this.id, event, #{details})"
+            else 
+                return "live.handleForm(this.id, event)"
+            end 
+        end 
+
+        #Render the element
```

Comment:
Please include a space between `#` and the comment, e.g.

```
# Render the form element.
```

The `returns` line is probably not needed.

Reply:
[072b9f9 ](https://github.com/socketry/live/pull/6/commits/072b9f9e06d1d20f1ef859dab661fc1295ee29e5)